book/2_modelling_advanced_ewatercycle/hbv_bmi.py

```diff
@@ -499,4 +499,4 @@ def load_var(ncfile: str | Path, varname: str) -> xr.DataArray:
         if data[varname].attrs['units'] == 'kg m-2 s-1':
             data[varname] = data[varname] * 24 * 3600 #mm/day
             #data[varname].attrs['units'] = 'mm d-1' TODO, fix.
```

Comment:
wouldn't hurt to fix this too, it seems OK to me, not sure why it says "TODO, fix".
```suggestion
            data[varname].attrs["units"] = "mm d-1"
```

Reply:
One small detail here is also that this can fail quietly, as the units are not explicitly checked. But it's probably OK here